pypika/terms.py

```diff
@@ -84,8 +84,13 @@ def between(self, lower, upper):
 
     def isin(self, arg):
         if isinstance(arg, (list, tuple, set)):
-            return ContainsCriterion(self, ListField([self._wrap(value) for value in arg]))
-        return ContainsCriterion(self, arg)
+            return ContainsCriterion(self, ListField([self._wrap(value) for value in arg]), isin=True)
+        return ContainsCriterion(self, arg, isin=True)
+
+    def isnotin(self, arg):
```

Comment:
Could you add an override for `__not__` in criterion which sets the negation boolean, currently named `isin`?  Also, more of a nitpick, I think this function name would sound nicer as just `notin`.